interfaces/IFlight.ts

```diff
@@ -12,4 +16,4 @@ export default interface IFlight {
     departure: string;
     arrival: string;
   }[];
-}
+}
```

Comment:
This is why I had prettier add newlines at the end. I find this red icon ugly.

Reply:
can't see the red icon